test/python/test_localjob.py

```diff
@@ -150,10 +149,8 @@ def intercepted_executor_for_localjob():
 def mocked_simulator_binaries():
     """Context to force binary-based simulators to think the simulators exist.
     """
-    from qiskit.backends.local import qasm_simulator_projectq
     with patch.object(path, 'exists', return_value=True, autospec=True),\
-            patch.object(path, 'getsize', return_value=1000, autospec=True),\
-            patch.object(qasm_simulator_projectq, 'CppSim', {}):
```

Comment:
@delapuente can you please double check this to see if the intent of your tests are preserved if we remove ProjectQ simulator here?

Reply:
Double checked, it is ok!